meilisearch.pkr.hcl

```
@@ -31,10 +31,35 @@ packer {
 }
 
 source "amazon-ebs" "debian" {
-  ami_name        = "${var.image_name}-${var.meilisearch_version}-${var.base-os-version}"
-  instance_type   = "t2.small"
-  region          = "us-east-1"
+  ami_name      = "${var.image_name}-${var.meilisearch_version}-${var.base-os-version}"
+  instance_type = "t2.small"
+  region        = "us-east-1"
+  ami_regions = [
+    "us-east-1",
+    "us-east-2",
+    "us-west-1",
+    "us-west-2",
+    "af-south-1",
+    "ap-east-1",
+    "ap-south-1",
+    "ap-northeast-2",
+    "ap-southeast-1",
+    "ap-southeast-2",
+    "ap-northeast-1",
+    "ca-central-1",
+    "eu-central-1",
+    "eu-west-1",
+    "eu-west-2",
+    "eu-south-1",
+    "eu-west-3",
+    "eu-north-1",
+    "me-south-1",
+    "sa-east-1",
```

```suggestion
    "ap-east-1",
    "ap-northeast-1",
    "ap-northeast-2",
    "ap-northeast-3",
    "ap-south-1",
    "ap-south-2",
    "ap-southeast-1",
    "ap-southeast-2",
    "ap-southeast-3",
    "ap-southeast-4",
    "ca-central-1",
    "cn-north-1",
    "cn-northwest-1",
    "eu-central-1",
    "eu-central-2",
    "eu-north-1",
    "eu-south-1",
    "eu-south-2",
    "eu-west-1",
    "eu-west-2",
    "eu-west-3",
    "me-central-1",
    "me-south-1",
    "sa-east-1",
    "us-east-1",
    "us-east-2",
    "us-gov-east-1",
    "us-gov-west-1",
    "us-west-1",
    "us-west-2"
```
Here is the updated regions list :)

---

Thanks ! But I think we don't have access to it instead we modify the it:
```
There are 6 Regions that are not enabled for this account:
ap-south-2
ap-southeast-3
ap-southeast-4
eu-south-2
eu-central-2
me-central-1
```
Did we make the modification on the AWS account before?

---

I've enabled the remaining ones into our account, it take some minutes in order to make it work :)

---

I think we can merge it now, and then check in the next release if all of the regions are available correctly!